README.md

```diff
@@ -35,7 +35,8 @@ A number of atlases are in development, but those available currently are:
 - [3D Edge-Aware Refined Atlases Derived from the Allen Developing Mouse Brain Atlases](https://doi.org/10.7554/eLife.61408) (E13, E15, E18, P4, P14, P28 & P56)
 - [Princeton Mouse Brain Atlas](https://brainmaps.princeton.edu/2020/09/princeton-mouse-brain-atlas-links) at 20 micron resolution
 - [Kim Lab Developmental CCF (P56)](https://data.mendeley.com/datasets/2svx788ddf/1) at 10 micron resolution with 8 reference images - STP, LSFM (iDISCO) and MRI (a0, adc, dwo, fa, MTR, T2)
-- [Blind Mexican Cavefish Brain Atlas](https://doi.org/10.7554/eLife.80777) at 2um resolution
+* [Blind Mexican Cavefish Brain Atlas](https://doi.org/10.7554/eLife.80777) at 2 micron resolution
+* [BlueBrain Barrel Cortex Atlas](https://doi.org/10.1162/imag_a_00209) at 10 and 25 micron resolution
```

Comment:
```suggestion
- [Blind Mexican Cavefish Brain Atlas](https://doi.org/10.7554/eLife.80777) at 2 micron resolution
- [BlueBrain Barrel Cortex Atlas](https://doi.org/10.1162/imag_a_00209) at 10 and 25 micron resolution
```

(Very, very) Slightly nicer to read in MarkDown, because consistent will bullet points above?